umm-spec-lib/src/cmr/umm_spec/umm_to_xml_mappings/dif10.clj

```diff
@@ -194,34 +200,20 @@
     [:Related_URL
      [:URL u/not-provided-url]]))
 
-(def collection-progress->dif10-dataset-progress
-  "Mapping from known collection progress values to values supported for DIF10 Dataset_Progress."
-  {"PLANNED" "PLANNED"
-   "ONGOING" "IN WORK"
-   "ONLINE" "IN WORK"
-   "COMPLETED" "COMPLETE"
-   "FINAL" "COMPLETE"})
-
-(def dif10-dataset-progress-values
-  "Set of Dataset_Progress values supported in DIF10"
-  (set (distinct (vals collection-progress->dif10-dataset-progress))))
-
 (defn- generate-dataset-progress
   "Return Dataset_Progress attribute by translating from the UMM CollectionProgress to one of the
   DIF10 enumerations. Defaults to generating a Dataset_Progress of IN WORK if translation cannot be
   determined."
   [c]
   (when-let [c-progress (when-let [coll-progress (:CollectionProgress c)]
-                          (str/upper-case coll-progress))]
-    [:Dataset_Progress (if (dif10-dataset-progress-values c-progress)
-                         c-progress
-                         (get collection-progress->dif10-dataset-progress c-progress "IN WORK"))]))
+                          (get coll-progress-mapping (string/upper-case coll-progress)))] 
```

Comment:
No, it's not.  The previous code was removed and changed to a when, so the relative spacing here is not a good reference.